packages/uploadthing/src/client.ts

```diff
@@ -33,9 +37,12 @@ function fetchWithProgress(
   });
 }
 
-const createRequestPermsUrl = (config: { url?: string; slug: string }) => {
-  const queryParams = `?actionType=upload&slug=${config.slug}`;
-
+const createRequestUrl = (config: {
+  url?: string;
+  slug: string;
+  actionType: ActionType;
+}) => {
+  const queryParams = `?actionType=${config.actionType}&slug=${config.slug}`;
```

Comment:
If we're going nit I'd say use some `URL` or `URLSearchParams` here instead of a raw string but thats very nit